### ISSUE TITLE
some rooms are shown as Empty room with lazy loading

### ISSUE THREAD

**Issue by 532910** [vector-im/riot-android#2671]
but room details shows two active members

**Comment by dva-re** [comment]
on latest dev build it is also appears =(

**Comment by non-Jedi** [comment]
For what it's worth, this is causing significant usability problems for my non-technical family members. It seems to happen only on 1-to-1 rooms with 2 different homeservers, but it isn't every room. Please chat with me @adam:thebeckmeyers.xyz if more details are needed or let me know if a rageshake would be helpful for this one.

**Comment by dkasak** [comment]
FWIW, it's happening to me in a 1-on-1 room between me and another person on the *same* homeserver too.

**Comment by r4dh4l** [comment]
I have this problem since the last and the current version of Riot-Android installed via [F-Droid](https://f-droid.org/en/packages/im.vector.alpha/) (currently 0.8.21 (82100) - added on 2019-01-03, don't remember the version before, maybe Version 0.8.20 (82000) - added on 2018-12-27 ).

**Comment by mvgorcum** [comment]
> In my case the affected 1on1-rooms contain only users from the same homeserver (which is not matrix.org).

I have the same issue. It's a single room that is affected.

Most notably it's a room that at some point was renamed, and the name was subsequently removed, causing the 1:1 room to revert back to the normal behavior (displaying the name of the other user in the room) without lazyloading.

**Comment by esackbauer** [comment]
> Most notably it's a room that at some point was renamed, and the name was subsequently removed, causing the 1:1 room to revert back to the normal behavior (displaying the name of the other user in the room) without lazyloading.

Exactly the same with me.

**Comment by r4dh4l** [comment]
> Most notably it's a room that at some point was renamed, and the name was subsequently removed, causing the 1:1 room to revert back to the normal behavior (displaying the name of the other user in the room) without lazyloading.

So providing a room name and removing it again solved the problem for you? Not working in my case, just tested. :/

**Comment by mvgorcum** [comment]
No, I'm saying that adding a name and removing it happened in the past. Which worked fine without LL.

**Comment by r4dh4l** [comment]
Okay, sry and thx for confirming. I don't remember if I renamed the affected rooms in the past. I doubt that I did because all of them were created as 1:1 rooms and there is no need to give a special name to such rooms.

**Comment by mehturt** [comment]
For me it happens when I change name or even avatar

**Comment by croulibri** [comment]
> 
> 
> I have this problem since the last and the current version of Riot-Android installed via [F-Droid](https://f-droid.org/en/packages/im.vector.alpha/) (currently 0.8.21 (82100) - added on 2019-01-03, don't remember the version before, maybe Version 0.8.20 (82000) - added on 2018-12-27 ).
> On Riot-Web/Desktop all Rooms are listed correctly.
> 
> > It seems to happen only on 1-to-1 rooms with 2 different homeservers
> 
> In my case the affected 1on1-rooms contain only users from the same homeserver (which is not matrix.org).

I face exactly the same situation.
Removing the lazy loading correct the problem as the room name take again the name of the correspondent.

**Comment by schildbach** [comment]
I had the "Empty room" problem too after I set my avatar. I just disabled "lazy loading" and this fixed this issue for me.
